README.md

```diff
@@ -14,4 +14,8 @@ Here's a very technical definition from [Wikipedia](https://en.wikipedia.org/wik
 ## Projects
 
 - [XState](https://github.com/davidkpiano/xstate)
+- [Kingly](https://brucou.github.io/documentation)
+- [Sismic](https://sismic.readthedocs.io)
+- [Elixir gen_statem](https://andrealeopardi.com/posts/connection-managers-with-gen_statem)
```

Comment:
Might be more fitting to put this in the write-ups section?

Reply:
Do you mean move `Elixir gen_statem` link to Reading section?

Because all of them (except the Elixir) are libraries for defining state machine.

Reply:
Yes I was referring to that link specifically. I figured it was more a write-up about how to apply state machines using that library, but I don't have super strong opinions on where it should go. Happy to just merge this as is if that's what you think!

Reply:
done